### DISCUSSION TITLE
Support for ED103TC2

### DISCUSSION THREAD

**Discussion by v11** [vroland/epdiy#359] [General]
Hi all

I have a ED103TC2 display. You can get it from [Waveshare](https://www.waveshare.com/product/displays/e-paper/epaper-1/10.3inch-e-paper.htm) but you can get it also from other suppliers. 

Specs:
10.3inch
1872×1404
16 Grey Scales

I have the datasheet (@martinberlin you have it too), but I can't share it here.

Is there anybody who could check it this display would be compatible with v7 and help me integrate it?

**Comment by martinberlin** [comment]
@vroland if I remember well you got working the 16 bit mode testing [with this 7.8 display](https://www.waveshare.com/product/displays/e-paper/7.8inch-e-paper-hat.htm) that has the same 40 pin connector and resolution right?

**Comment by Dreadmond** [comment]
Hello. Sorry to resurrect this thread. I have a board arriving soon - I will let everyone know if I am able to get it to work.

**Reply by jantielens** [reply]
Would be very interested to hear how it went and how the display quality/contrast is!

**Reply by dugdathug** [reply]
@Dreadmond Any updates?  I just got my units today

**Comment by martinberlin** [comment]
Be aware that you need to invert signals with this display right?
Just check at the schematics and PCB vs display 1-40 pins. You will need a 40 to 40 pins connector plus a type B 0.5 pitch flex cable with connecting pins below plus output pins above (That inverts the 1-40 to 40-1)

**Comment by Dreadmond** [comment]
Hello all,

I could get it to work with the settings discussed above, but afraid I struggled to get the display to perform to a standard that I would like.

Essentially, it did not display the full gamut of greyscale options despite me trying multiple waveforms. Images appeared degraded as a result.

And yes, Martin is correct that the signals need to be inverted.

I may well look into this again in the future, will update everyone if I have any success.

**Reply by Herasium** [reply]
If you get something, I'll be glad to hear about it. I got an ED103TC2 display, and I still don't know if I should use epdiy or a standard it8951 tcon to drive it.

**Reply by dugdathug** [reply]
> If you get something, I'll be glad to hear about it. I got an ED103TC2 display, and I still don't know if I should use epdiy or a standard it8951 tcon to drive it.

I went with XIAO ePaper Display Board EE03 from Seeed and it worked perfectly for me.
